tequilapi/endpoints/list_identities.go

```diff
@@ -0,0 +1,30 @@
+package endpoints
+
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/identity"
+	"github.com/mysterium/node/service_discovery/dto"
+	"github.com/mysterium/node/tequilapi/utils"
+)
+
+type identityList struct {
+	Identities []dto.Identity `json:"identities"`
+}
+
+type listIdentitiesEndpoint struct {
+	listIdentities func() []dto.Identity
```

Comment:
I would inject `IdentityManager` as dependency

Reply:
now injected on IdentityHandlers construction.
or are you suggesting to inject it for every api call separately?
we now only have `list` but there might be others..